.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - UX Audit
```

Comment:
Any other labels to exclude?

Reply:
Maybe the Up Next labels?  To avoid sort of active work?

Reply:
that makes sense

Reply:
What about "bug" but i guess we usually don't ignore bug issues for 60 days.
What about "Priority: High"? We don't use this much but again we shouldn't ignore high priority issue for 60 days.